ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/scribble/v1/scribble/scribble.html

```diff
@@ -0,0 +1,92 @@
+              <input type="text" class="cmp-adaptiveform-scribble__keyboard-sign-box" name="signatureText" placeholder="${scribble.placeHolder}" aria-label="${'signature text' @ i18n}" tabindex="-1">
+            </div>
+            <canvas class="cmp-adaptiveform-scribble__geocanvas" width="0" height="0" aria-label="${'geolocation' @ i18n}" ></canvas>
+          </div>
+          <div class="cmp-adaptiveform-scribble__controlpanel">
+            <div class="cmp-adaptiveform-scribble__controls">
+              <button type="button" class="cmp-adaptiveform-scribble__control-brush cmp-adaptiveform-scribble__button" data-cmp-scribble-button="brushes" tabindex="0" aria-label="${'brushes' @ i18n}"></button>
+              <div class="cmp-adaptiveform-scribble__brushlist" aria-label="${'brush size selection' @ i18n}" style="display: none;"></div>
+              <button type="button" class="cmp-adaptiveform-scribble__control-clear cmp-adaptiveform-scribble__button" data-cmp-scribble-button="clearSign" tabindex="0" aria-label="${'clearSign' @ i18n}" ></button>
+              <button type="button" class="cmp-adaptiveform-scribble__control-geo cmp-adaptiveform-scribble__button" data-cmp-scribble-button="geolocation" tabindex="0" aria-label="${'geolocation' @ i18n}" ></button>
+              <button type="button" class="cmp-adaptiveform-scribble__control-text cmp-adaptiveform-scribble__button" data-cmp-scribble-button="textSign" tabindex="0" aria-label="${'textSign' @ i18n}"></button>
+              <div class="cmp-adaptiveform-scribble__control-message"></div>
+            </div>
+            <div class="cmp-adaptiveform-scribble__controlpanel__controls">
+              <button class="cmp-adaptiveform-scribble__close-button" data-cmp-scribble-button="close" tabindex="0" type="button" aria-label="${'Close' @ i18n}">${"Close" @ i18n}</button>
+              <button class="cmp-adaptiveform-scribble__save-button" data-cmp-scribble-button="save" tabindex="0" type="button" aria-label="${'Save' @ i18n}">${"Save" @ i18n}</button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="cmp-adaptiveform-scribble__clearsign-container">
+        <h2 class="cmp-adaptiveform-scribble__clearsign-title">${"Erase Current Signature?" @ i18n}</h2>
+        <div class="cmp-adaptiveform-scribble__clearsign-content" class="msgBoxTypenull">
+          <div class="cmp-adaptiveform-scribble__clearsign-message">
+            ${"This will permanently remove the signature you've drawn. Do you wish to proceed and begin again?" @ i18n}
+          </div>
+          <div class="cmp-adaptiveform-scribble__clearsign-panel">
+            <button type="button" id="msgBox_No" class="cmp-adaptiveform-scribble__button--secondary">${"Cancel" @ i18n}</button>
+            <button type="button" id="msgBox_Yes" class="cmp-adaptiveform-scribble__button--primary">${"Clear" @ i18n}</button>
```

Comment:
It generally recommended to also include the block with the modifier in the HTML structure, so this should be like this

class="cmp-adaptiveform-scribble__button cmp-adaptiveform-scribble__button--primary"